main.rb

```diff
@@ -38,11 +59,17 @@ def show_message(msg)
 
 flag_value = client.variation(feature_flag_key, context, false)
 
-show_message "Feature flag '#{feature_flag_key}' is #{flag_value} for this context"
+show_flag_message(feature_flag_key, flag_value)
+
+if ci
+  exit 0
+end
```

Comment:
Ruby has this nice little shorthand you can use.

```suggestion
exit 0 if ci
```